content/en/account_management/billing/profiler.md

```diff
@@ -79,7 +79,7 @@ In this scenario, we aggregate all containers across all hosts (so two hosts, 10
 | Total          |            |                                                                                                 | $24 + $4      | **$28 per month**    |
 
 
-## FAQs
+## FAQ
```

Comment:
Oooh is this something we could potentially add a vale rule for styling/word choice to make sure any new sections are always FAQ in case someone accidentally makes it "FAQs"?